Immunodeficiency touch ups

throwawayuseless opened the pull request:
## About The Pull Request
Fixes and balancing for my new quirk and chem woo:
FIXES: Fixes a typo, also prevents species that are inorganic or that are immune to viruses from picking immunodeficient. Also i forgot to remove the default taste from sepsisillin so it was detectable if ingested.

BALANCING: Made immunodeficient -6 instead of -10. Made immunodeficiency effects prevent you from naturally recovering from diseases. Cures still work, though, as was already the case, they'll work slower if you're immunodeficient.
## Why It's Good For The Game
Typo bad, magical free quirk points for plasmamen bad.  Long-duration stealth poisons with identifiable tastes are terrible long duration stealth poisons.

also, -6 brings it in line with other "highly detrimental but only in a specific niche" quirks because -10 was too much oops. Natural disease recovery meant that even though immunodeficiency made a disease progress faster it would eventually taper down to stage 1 (due to the timer based virus peaking system) and sit there doing nothing until fixed or until you got really lucky and it self cured. Really stupid, shouldn't do that.
## Changelog
:cl:
fix: Plasmamen cannot take immunodeficiency for free quirk points.
fix: Sepsisillin is actually stealthy now (I forgot to remove its taste)
fix: A single typo.
balance: Immunodeficiency effects now prevent natural unaided disease recovery. 
balance: Immunodeficiency now -6 instead of -10.
/:cl:

throwawayuseless committed:
Inverty Magurdy

throwawayuseless committed:
Merge branch 'tgstation:master' into stupid-reagent-funnies

throwawayuseless committed:
Add sepsisillin

It's like spaceacillin but evil.

throwawayuseless committed:
Update _traits.dm

throwawayuseless committed:
add the trait woweee

throwawayuseless committed:
zmobie

throwawayuseless committed:
Update _MobProcs.dm

throwawayuseless committed:
Update _disease.dm

throwawayuseless committed:
Update _disease.dm

throwawayuseless committed:
Update burns.dm

throwawayuseless committed:
Update alien_embryo.dm

throwawayuseless committed:
Update burns.dm

throwawayuseless committed:
Update _disease.dm

throwawayuseless committed:
Create immunodeficiency

throwawayuseless committed:
Rename immunodeficiency to immunodeficiency.dm

throwawayuseless committed:
Update immunodeficiency.dm

throwawayuseless committed:
Update immunodeficiency.dm

throwawayuseless committed:
add spaceacillin pill

throwawayuseless committed:
Update pillbottles.dm

throwawayuseless committed:
Update immunodeficiency.dm

throwawayuseless committed:
Update immunodeficiency.dm

throwawayuseless committed:
Update pillbottles.dm

throwawayuseless committed:
Update pill.dm

Commit:
Bugfixes and testing

throwawayuseless committed:
Merge branch 'tgstation:master' into stupid-reagent-funnies

Commit:
Merge branch 'stupid-reagent-funnies' of https://github.com/throwawayuseless/thatguywhoprsthings into stupid-reagent-funnies

throwawayuseless committed:
Remove whatever that was that was there in genesis_call.dme

throwawayuseless committed:
Removed the faster spread effects.

throwawayuseless committed:
Merge branch 'master' into stupid-reagent-funnies

throwawayuseless committed:
Update pillbottles.dm

throwawayuseless committed:
ME WHEN I INDENTATION ERROR

throwawayuseless committed:
fixed it woowowow

throwawayuseless committed:
Merge branch 'tgstation:master' into stupid-reagent-funnies

throwawayuseless committed:
No immunodeficient plasmamen

throwawayuseless committed:
Update impure_medicine_reagents.dm

LemonInTheDark commented:
if you are changing procs I need you to at least be able to compile it locally. please bug me and I can walk you through setting that up if required

throwawayuseless commented:
> if you are changing procs I need you to at least be able to compile it locally. please bug me and I can walk you through setting that up if required

I've got a localhost and i'll post testing results soon. should have marked this as draft lmao sorry

throwawayuseless committed:
oops

i did it again

Co-authored-by: tonty <39193182+tontyGH@users.noreply.github.com>

btcbuster commented:
i think we should just give plasmamen free quirk points at this rate

throwawayuseless committed:
Merge branch 'tgstation:master' into stupid-reagent-funnies

throwawayuseless committed:
Species check

throwawayuseless commented:
> i think we should just give plasmamen free quirk points at this rate

they **are** kinda weak normally

throwawayuseless committed:
-6 instead of -10

brings immunodeficiency in line with other "very detrimental, but only in their niche" quirks because -10 is kind of a lot of points

throwawayuseless committed:
Immunodeficiency prevents natural recovery. Actual cures are unchanged.

I'd have to imagine sleeping off a case of the flu is much harder with far future space AIDS.

throwawayuseless committed:
Merge branch 'master' into stupid-reagent-funnies

SmArtKar commented:
> > if you are changing procs I need you to at least be able to compile it locally. please bug me and I can walk you through setting that up if required
> 
> I've got a localhost and i'll post testing results soon. should have marked this as draft lmao sorry

You're still uploading webedits, and we don't need testing results - we need assurance that you're testing all of your changes.

LemonInTheDark commented:
You will suffer for your sins
